packages/extension-base/src/background/handlers/State.ts

```diff
@@ -60,14 +60,9 @@ interface SignRequest extends Resolver<ResponseSigning> {
 let idCounter = 0;
 
 const WINDOW_OPTS: chrome.windows.CreateData = {
-  // This is not allowed on FF, only on Chrome - disable completely
-  // focused: true,
-  height: 621,
-  left: 150,
-  top: 150,
+  focused: true,
```

Comment:
Focused is supported: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/windows/create